src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/BuildRazorCommand.cs

```diff
@@ -1,85 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.AspNetCore.Blazor.Build.Core.RazorCompilation;
-using Microsoft.AspNetCore.Blazor.Razor;
-using Microsoft.Extensions.CommandLineUtils;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-
-namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
-{
-    internal class BuildRazorCommand
```

Comment:
These are not needed anymore. These are pretty similar to what's in the Razor SDK